juice-examples/mnist-image-multiclass-classification/src/main.rs

```diff
@@ -112,182 +118,89 @@ fn main() {
                 panic!()
             }
     } else if args.cmd_fashion {
-        #[cfg(all(feature = "cuda"))]
-            run_fashion(
+        run_mnist(
+            MnistType::Fashion,
             args.arg_model_name,
             args.arg_batch_size,
             args.arg_learning_rate,
             args.arg_momentum,
         );
-        #[cfg(not(feature = "cuda"))]
```

Comment:
:tada:

Reply:
Hah, you can thank Paul for a lot of that. I took a few liberties from his MNIST Native example.

Reply:
The :tada; was more for cleaning it finally up so we can remove some disclaimers, thanks to @paulkass (:heart:) for the heavy lifting!